CMakeLists.txt

```diff
@@ -198,6 +202,14 @@ add_executable(pose_calibrator ${POSE_CALIBRATOR_SRC})
 add_dependencies(pose_calibrator ${PROJECT_NAME}_gencpp)
 target_link_libraries(pose_calibrator ${MCPTAM_LIBS} mcptam_functions)
 
+if(BUILD_TEST)
+  MESSAGE("adding files for unit testing...")
```

Comment:
Can we make the message very clear, where it is coming from, what tests are being built? Not just adding files.